src/Pi.cpp

```diff
@@ -1615,9 +1619,35 @@ void Pi::SetMouseGrab(bool on)
 void Pi::DrawPiGui(double delta, std::string handler)
 {
 	PROFILE_SCOPED()
+	bool debug = false;
```

Comment:
This debug/profiling code needs removing

Reply:
I did it that way to get it compiled: reason is that `#ifdef` clause don't check code
thus after years you have code which may not compile...

...I also would let that code "in" so next time an optimization is needed then
I (or some other) will get the same numbers.

@impaktor : that code, if I follow the rules you explained , must be fixed in
the first commit or I'm wrong?

Reply:
Is there a reason that you didn't just use the builtin profiling which is already there? Meaning the call to `PROFILE_SCOPED()` above your own profiling code?

Reply:
In general, is easier to collect times because it print some message and you only copy-paste these ones...

In particular, I follow the instructions you gave in your previous PR but I can't get anything.

But you're right: I will remove this code by removing the related commits.